### PR TITLE
add core.gen.tr to community/commercial-support

### PR DESCRIPTION
**Pull request by evrim**
Let me know if there is a typo, have no means of linting ;)

best,
evrim.

### CODE REVIEW

File: community/commercial-support.toml [hunk 1]
```diff
@@ -137,3 +137,13 @@ description = """
 flox is a platform for building, sharing, and collaborating across the entire
 development lifecycle.
 """
+
+[[commercial-provider]]
+name = "core.gen.tr"
+url = "https://core.gen.tr"
+locations = ["Remote" "Istanbul"]
```
Comment (figsoda) [inline community/commercial-support.toml:144]:
```suggestion
locations = ["Remote", "Istanbul"]
```

Comment (evrim) [reply community/commercial-support.toml]: OK, added the missing comma, thnx!

**Review by garbas — CHANGES_REQUESTED** [review]
https://core.gen.tr url is not working for me.

**Comment by evrim** [comment]
ping @garbas

**Comment by garbas** [comment]
Hi @evrim!

I'm a little hesitant to merge this PR. Here are the reasons:
- The url points to a blog
- No where on the website there is a mention that there you are providing Nix services.
- There is only one blog post regarding Nix

The page commercial provides should include links with commercial providers for Nix users and I don't think your URL meets the criteria. I will bring this up at the next Marketing meeting. We should also write down the criteria.

I'll be back with the answer in 10 days.

**Comment by tomberek** [comment]
> page commercial provides should include links with commercial providers for Nix users and I don't think your URL meets the criteria. I will bring this up at the next Marketing meeting. We should

I agree with @garbas 's assessment.

**Comment by evrim** [comment]
I am afraid I am not here to satisfy your egos. If you change the rules after the game starts, nobody will have any trust in the rules or the system. The time will come that this record will be used against who do just that. Close the issue if you are trying to mess around with me and my enterprise.

best,
evrim.

**Comment by tomberek** [comment]
The commercial support page is not meant to be a blog aggregator. The proposed link is a blog and it is not clear how someone looking for commercial support should contact you. We'd be happy to include or showcase specific Nix content via announcements or cross-posting. Closing. Feel free to reopen if the situation changes.

**Comment by evrim** [comment]
Thnx for the confirmation.

**Comment by evrim** [comment]
re-open, re-eval request, check out https://www.core.gen.tr

**Comment by evrim** [comment]
ping @tomberek @garbas

**Pull request closed without merging**